mis_builder_budget/models/mis_budget_by_account_item.py

```diff
@@ -24,15 +24,10 @@ class MisBudgetByAccountItem(models.Model):
         store=True,
         currency_field="company_currency_id",
     )
-    company_id = fields.Many2one(
-        "res.company",
-        related="account_id.company_id",
-        readonly=True,
-        store=True,
-    )
+    company_id = fields.Many2one("res.company", compute="_compute_company_id")
     company_currency_id = fields.Many2one(
         "res.currency",
-        related="account_id.company_id.currency_id",
+        related="account_id.company_currency_id",
```

Comment:
This is not correct, I think, because account_id.company_currency_id is non stored and context dependent.